core/store/migrate/migrations/0250_ccip_token_prices_fix.sql

```diff
@@ -0,0 +1,49 @@
+-- +goose Up
+
+-- We need to re-create tables from scratch because of the unique constraint on tokens and chains selectors
+DROP TABLE ccip.observed_token_prices;
+DROP TABLE ccip.observed_gas_prices;
+
+CREATE TABLE ccip.observed_token_prices
+(
+    chain_selector NUMERIC(20, 0) NOT NULL,
+    token_addr     BYTEA          NOT NULL,
+    token_price    NUMERIC(78, 0) NOT NULL,
+    created_at     TIMESTAMPTZ    NOT NULL DEFAULT NOW(),
+    PRIMARY KEY (chain_selector, token_addr)
```

Comment:
Add index for created_at (if we're going to continue with using it in query as suggested earlier in the PR)